streamdl_proto_srv.py

```
@@ -55,26 +58,61 @@
+class HealthHandler(BaseHTTPRequestHandler):
+    def do_GET(self):
+        if self.path == "/health":
+            self.send_response(200)
+            self.send_header("Content-type", "text/plain")
+            self.end_headers()
+            self.wfile.write(b"OK")
+        else:
+            self.send_response(404)
+            self.end_headers()
+
+    def log_message(self, format, *args):
```

Ruff (0.15.9)

[error] Function argument `format` is shadowing a Python builtin

(A002)